pkg/volume/cinder/cinder.go

```diff
@@ -35,11 +36,13 @@ import (
 
 // This is the primary entrypoint for volume plugins.
 func ProbeVolumePlugins() []volume.VolumePlugin {
-	return []volume.VolumePlugin{&cinderPlugin{nil}}
+	return []volume.VolumePlugin{&cinderPlugin{}}
 }
 
 type cinderPlugin struct {
 	host volume.VolumeHost
+	// Guarding SetUp and TearDown operations
+	volumeGuard keymutex.KeyMutex
```

Comment:
Rename to "lock" or "mutex"?  I've seen those two names throughout the codebase. I've used both in the pv controllers, but should probably standardize on one.

I prefer "mutex" (since that's what the thing actually is) and I will be sure to change "lock" to "mutex" in the controllers as we refactor them.